src/analysisd/format/to_json.c

```diff
@@ -153,6 +153,83 @@ char* Eventinfo_to_jsonstr(const Eventinfo* lf, bool force_full_log)
                 cJSON_Delete(mitre_technique_array);
             }
         }
+        if(lf->generated_rule->mitre_technique_id && lf->generated_rule->mitre_tactic_id) {
```

Comment:
I think this will be better:

```c
if(lf->generated_rule->mitre_technique_id && lf->generated_rule->mitre_tactic_id) {

...

} else if (lf->generated_rule->mitre_id) {

...

}
```